docs/getting-started/clients/mobile/android/index.md

```diff
@@ -58,11 +58,56 @@ To set up a new virtual Android device for debugging:
 
 ![Android SDK configuration](android-sdk.png)
 
-## Testing and Debugging using the Android Emulator
+## Testing and Debugging
+
+### Using the Android Emulator
 
 In order to access `localhost:<port>` resources in the Android Emulator when debugging using Xamarin
 studio on your mac natively, you'll need to configure the endpoint addresses using
 `<http://10.0.2.2:<port>`\> in order to access `localhost`, which maps the Android proxy by design.
 
 [xamarin-vs]: https://learn.microsoft.com/en-us/xamarin/android/get-started/installation/android-sdk
 [android-studio]: https://developer.android.com/studio/releases/platforms
+
+### Using Server Tunneling
+
+Instead of configuring your device or emulator, you can instead use a
+[proxy tunnel to your local server](../../../server/tunnel.md) and have your app connect to it
+directly.
+
+### Push Notifications
+
+The default configuration for the Android app is to register itself to the same environment as
+Bitwarden's QA Cloud. This means that if you try to debug the app using the production endpoints you
+won't be able to receive Live Sync updates or Passwordless login requests.
+
+<bitwarden> 
+So, in order to receive notifications while debugging, you have two options:
+- QA Cloud endpoints for the Api and Identity, or 
```

Comment:
```suggestion
- Use QA Cloud endpoints for the Api and Identity, or 
```

Reply:
In addition, I think you'll need a blank line before your bulleted list starts; it looks like the markdown isn't formatting it correctly right now.

Reply:
yup it was broken, fixed it as well.